README.md

```diff
@@ -104,6 +104,7 @@ Depending on the scenario in which you utilize your proxy, the proxy container e
 1. 80: Standard web traffic (HTTP)
 2. 443: Standard web traffic, encrypted (HTTPS)
 3. 5222: Jabber protocol traffic (WhatsApp default)
+4. 587 or 7777: *.whatsapp.net traffic including media
 
 There are also ports configured which accept incoming [proxy headers](https://www.haproxy.com/blog/use-the-proxy-protocol-to-preserve-a-clients-ip-address/) (version 1 or 2)
```

Comment:
haproxy.com/blog/use-the-proxy-protocol-to

Reply:
Ok